### PR TITLE
Hearing Prep user Reader access

### PR DESCRIPTION
**Pull request by sharonwarner**
This PR 
- Gives reader access to hearings users
- Updates the hearing role to 'Hearing Prep'

Connects #2747 

To validate:
- [x] Select the Hearing Prep user
- [x] Confirm you can access Reader (may have to enable Reader's Feature Toggle)
- [x] Confirm you can access Hearing Prep

screen shot 2017-08-03 at 5 50 30 pm
screen shot 2017-08-03 at 6 03 02 pm

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -9,11 +9,12 @@ class User < ActiveRecord::Base
...
   FUNCTIONS = ["Establish Claim", "Manage Claim Establishment", "Certify Appeal",
-               "CertificationV2", "Reader", "Hearings"].freeze
+               "CertificationV2", "Reader", "Hearing Prep"].freeze
 
-  # Because of the funciton character limit, we need to also alias some functions
+  # Because of the function character limit, we need to also alias some functions
   FUNCTION_ALIASES = {
-    "Manage Claims Establishme" => ["Manage Claim Establishment"]
+    "Manage Claims Establishme" => ["Manage Claim Establishment"],
+    "Hearing Prep" => ["Reader"]
```
Comment (kierachell) [inline app/models/user.rb]: HAH! Nice!

Comment (mdbenjam) [inline app/models/user.rb]: Is this only one way? Does this give Reader access to Hearings Prep?

Comment (sharonwarner) [inline app/models/user.rb]: Yep, just one way.

Comment (kierachell) [inline app/models/user.rb]: If it was both, what's the point of separating the roles then :D

**Pull request merged**